docs/general/events.md

```diff
@@ -43,18 +33,12 @@ that identify the class of Events but not the instance of the Event.
 <!-- semconv event -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `event.domain` | string | The domain identifies the business context for the events. [1] | `browser` | Required |
-| `event.name` | string | The name identifies the event. | `click`; `exception` | Required |
-
-**[1]:** Events across different domains may have same `event.name`, yet be unrelated events.
-
-`event.domain` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+| `event.name` | string | Uniquely identifies the event. | `browser.mouse.click`; `browser.exception` | Required |
```

Comment:
```suggestion
| `event.name` | string | Identifies the class / type of event. | `browser.mouse.click`; `browser.exception` | Required |
```
The phrasing of "Uniquely identifies" reads like this could be some sort of UUID type identifier which is unique for every distinct record.